src/crystal/system/win32/event_loop_iocp.cr

```diff
@@ -0,0 +1,70 @@
+require "crystal/system/print_error"
+
+module Crystal::EventLoop
+  @@queue = Deque(Fiber).new
+
+  private def self.loop_fiber
```

Comment:
The EventLoop API changed a lot with MT. There is no more loop fiber, and less methods.